spec/presenters/queries/grouped_content_and_links_spec.rb

```diff
@@ -0,0 +1,93 @@
+require "rails_helper"
+
+module Presenters
+  module Queries
```

Comment:
You can do `RSpec.describe Presenters:: Queries::GroupedContentAndLinks` so it doesn't need the deep nesting.